storage/src/vespa/storage/visiting/reindexing_visitor.h

```diff
@@ -20,7 +20,7 @@ class ReindexingVisitor : public Visitor {
     ~ReindexingVisitor() override = default;
 
 private:
-    void handleDocuments(const document::BucketId&, std::vector<spi::DocEntry::UP>&, HitCounter&) override;
+    void handleDocuments(const document::BucketId&, DocEntryList&, HitCounter&) override;
```

Comment:
Consider updating the parameter type signature in the .cpp file as well

Reply:
Done